content/docs/guides/components/modeldb.md

```diff
@@ -0,0 +1,98 @@
++++
+title = "ModelDB"
+description ="ModelDB - A system to manage machine learning models"
+weight = 10
+toc = true
+[menu]
+[menu.docs]
+  parent = "components"
+  weight = 5
++++
+
+## Using ModelDB
+
+After ModelDB is deployed using kfctl.sh and the modeldb-db, modeldb-frontend and modeldb-backend pods are running - 
+
+1)  Port forward the modeldb-backend pod to port 6543 - 
+
+```
+kubectl get pods -n kubeflow # Find your modeldb-backend pod
+kubectl port-forward [modeldb-backend pod] 6543:6543 -n kubeflow
```

Comment:
With the latest kubectl you should be able to forward to a service
kubectl port-forward service/argocd-server 8080:443

But there will be an ambassador route
https://github.com/kubeflow/kubeflow/blob/master/kubeflow/modeldb/modeldb.libsonnet#L141
(The route will be /modeldb /katib is a bug). 

So we could point folks at those directions.

Reply:
You shouldn't need to port-forward for modeldb-backend; that's only required to access the frontned.

Reply:
are you sure, bcoz when I run my model python script without it gives me this error - thrift.transport.TTransport.TTransportException: Could not connect to any of [('::1', 6543, 0, 0), ('127.0.0.1', 6543)]